pkgs/development/libraries/science/chemistry/mmtf-cpp/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "mmtf-cpp";
-  version = "1.0.0";
```

Comment:
`mmtf-cpp` has been removed from the avogadro package (see above). Why its update then in this PR?

Reply:
The mmtf-cpp package was automatically updated by the original PR and the update itself is completely valid. The problem is that avogadrolibs is not compatible with the new mmtf-cpp version. The next avogadrolibs release will probably fix this, for the moment I think it is OK to update all three packages and disable mmtf-cpp in avogadrolibs temporarily.

Reply:
Since it removes functionality from avogadro, maybe we just keep the old mmtf-cpp around for now? Avogadro seems to be the only user of this library.

Reply:
OK, then let's use this version.